sharktank/tests/evaluate/perplexity_torch_test.py

```diff
@@ -8,10 +8,9 @@
 import pytest
 import json
 
+from sharktank.utils.testing import longrun
```

Comment:
I will add this change in my other PR, we can remove this here, if not using perplexity.

Reply:
I revered this change.